third_party/2and3/flask/globals.pyi

```diff
@@ -0,0 +1,15 @@
+# Stubs for flask.globals (Python 3.6)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .app import Flask
+from .wrappers import Request
+from typing import Any
+from werkzeug.local import LocalStack
+
+_request_ctx_stack: LocalStack
+_app_ctx_stack: LocalStack
+current_app: Flask
+request: Request
```

Comment:
Similar to above happens few times with `Request`, I am however not sure how common is the pattern of setting additional fields on `Request` objects.

Reply:
This one is valid, however it should probably be fixed on werkzeug's `Request`. I'll see what I can do.